packages/graphql-types-generator/src/flow-modern/__tests__/__snapshots__/codeGeneration.ts.snap

```diff
@@ -1,9 +1,10 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`Flow codeGeneration fragment spreads with inline fragments 1`] = `
-Object {
-  "__generated__/HeroName.js": FlowGeneratedFile {
-    "fileContents": "
+exports[`Flow codeGeneration fragment spreads with inline fragments 1`] = `"__generated__/HeroName.js"`;
+
+exports[`Flow codeGeneration fragment spreads with inline fragments 2`] = `
+FlowGeneratedFile {
+  "fileContents": "
 
```

Comment:
Are all the changes in this snap intended? I only see a regex replace of the original snapshot for windows. It should not have any changes to the original ones.

Reply:
Yes. Previously we were testing the file path and file contents all in one snapshot. After this change, these will be 2 different snapshots and the file path will be converted to Unix style for comparison.